src/Rector/Contrib/PHPUnit/SpecificMethod/AssertIssetToSpecificMethodRector.php

```diff
@@ -78,30 +71,54 @@ public function isCandidate(Node $node): bool
         /** @var Isset_ $issetNode */
         $issetNode = $firstArgumentValue;
 
-        return $issetNode->vars[0] instanceof PropertyFetch;
+        return in_array(get_class($issetNode->vars[0]), [
```

Comment:
Could you assign type to own variable? There is lot going on already

Reply:
>Could you assign type to own variable?

What do you mean? I didn't get it :disappointed:

Reply:
```php
in_array(get_class($issetNode->vars[0]),
```
This is like 4 operatios (inlined cyclomatic complexity) to read. 

This would be easier:

```php
$variableNodeClass = get_class($issetNode->vars[0]);
in_array($variableNodeClass, [...]);
```

Reply:
Done. Ready to merge?